src/polymer/dom-module-scanner.ts

```diff
@@ -87,11 +93,20 @@ export class DomModuleScanner implements HtmlScanner {
 
     await visit((node) => {
       if (isDomModule(node)) {
+        const template = dom5.query(node, dom5.predicates.hasTagName('template'));
+        let slots: Slot[] = [];
+        if (template) {
+          slots = dom5.queryAll(
+            treeAdapters.default.getTemplateContent(template),
+            dom5.predicates.hasTagName('slot')
+          ).map(s => new Slot(dom5.getAttribute(s, 'name') || ''));
```

Comment:
Oh nice catch! Can confirm that a slot name of `''` is the same as not providing a name attribute at all.